src/aarch64/ffi.c

```diff
@@ -31,6 +31,10 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.  */
 #include <windows.h> /* FlushInstructionCache */
 #endif
 
+#if __has_feature(ptrauth_calls)
```

Comment:
Thanks. I just noticed this caused a  build failure here: https://travis-ci.org/github/libffi/libffi/jobs/692670895#L652

Reply:
Cool!  I love CI that works =)  I've updated the title to make it more clear that I'm soliciting feedback on what you think is interesting to upstream rather than merge this in wholesale.  Parts that you find interesting, I'll definitely work to get integrated with autotools builds and cross-platform (as best I can at least) and spin off separate PRs for isolated changes.

Reply:
I put a lot of effort into getting the CI to work for lots of platforms, even implementing a new tool (https://rl.gl , you can see the libffi policy @ https://github.com/libffi/rlgl-policy/blob/master/XFAIL ), HOWEVER one big gap is around iOS testing.  I've never really participated in the iOS ecosystem.  If there was some way to run the testsuite on travis using a simulator, that would be wonderful.  Do you know if this is possible?